okl/plugins/lowMach.okl

```diff
@@ -79,7 +80,7 @@
           const dfloat dtdz = vgeo[gid + p_TZID * p_Np];
           const dfloat JW   = vgeo[gid + p_JWID * p_Np];
 
-          const dfloat iRhoCpT = 1 / (RHOCP[id] * Q[id]);
+		  const dfloat alpha = expansionCoeff[id]/RHOCP[id];
```

Comment:
You didn't introduce a flag if computing alpha is required or not.

Reply:
ok, i add this flag now